### ISSUE TITLE
Metrics mae

### ISSUE THREAD

**Issue by swadhin-** [SciSharp/TensorFlow.NET#790]
System.NotImplementedException
  HResult=0x80004001
  Message=
  Source=Tensorflow.Keras
  StackTrace:
   at Tensorflow.Keras.Engine.MetricsContainer._get_metric_object(String metric, Tensor y_t, Tensor y_p)
   at Tensorflow.Keras.Engine.MetricsContainer.<>c__DisplayClass9_0.<_get_metric_objects>b__0(String x)
   at System.Linq.Enumerable.SelectArrayIterator`2.ToArray()
   at System.Linq.Enumerable.ToArray[TSource](IEnumerable`1 source)
   at Tensorflow.Keras.Engine.MetricsContainer._get_metric_objects(String[] metrics, Tensor y_t, Tensor y_p)
   at Tensorflow.Keras.Engine.MetricsContainer.Build(Tensor y_true, Tensor y_pred)
   at Tensorflow.Keras.Engine.MetricsContainer.update_state(Tensor y_true, Tensor y_pred, Tensor sample_weight)
   at Tensorflow.Keras.Engine.Model.train_step(Tensor x, Tensor y)
   at Tensorflow.Keras.Engine.Model.train_step_function(OwnedIterator iterator)
   at Tensorflow.Keras.Engine.Model.FitInternal(Int32 epochs, Int32 verbose)
   at Tensorflow.Keras.Engine.Model.fit(NDArray x, NDArray y, Int32 batch_size, Int32 epochs, Int32 verbose, Single validation_split, Boolean shuffle, Int32 initial_epoch, Int32 max_queue_size, Int32 workers, Boolean use_multiprocessing)
   
[](https://github.com/SciSharp/TensorFlow.NET/blob/b54cbaa772cdd791155e3aeac90b45c656868a22/src/TensorFlowNET.Keras/Engine/MetricsContainer.cs)
   
   Hello,  Just encountered this error. From what I can see apart from **acc** all the other metrics are not implemented. I need to use **mae** to compete a model. Is there any plan to  implement it ?

**Comment by swadhin-** [comment]
[MetricsContainer.cs](https://github.com/SciSharp/TensorFlow.NET/blob/b54cbaa772cdd791155e3aeac90b45c656868a22/src/TensorFlowNET.Keras/Engine/MetricsContainer.cs)
